doc/releases/changelog-dev.md

```diff
@@ -26,6 +26,11 @@
 
 <h3>Deprecations</h3>
 
+* `qml.drawer.CircuitDrawer`, `qml.drawer.CHARSETS`, `qml.drawer.drawable_grid`, `RepresentationResolver`,
```

Comment:
Wonder if we should have a short summary sentence here and then separately list each class/method removed. Also, would this entry rather belong to Breaking Changes? Code that uses any of this infrastructure would break.